compiler/modules/front/codeGen.bal

```diff
@@ -322,6 +322,14 @@ function codeGenExpr(CodeGenContext cx, bir:BasicBlock bb, Scope? scope, Expr ex
             bb.insns.push(insn);
             return [result, nextBlock];
         }
+        var {bitwiseOp: op, left, right} => {
```

Comment:
I like to add a space after `{` and before `}` when braces are used in a single line.